src/Dlp/V2beta1/Gapic/DlpServiceGapicClient.php

```diff
@@ -384,21 +384,166 @@ public function __construct($options = [])
+     *     $inspectConfig = new InspectConfig();
+     *     $items = [];
+     *     $response = $dlpServiceClient->deidentifyContent($deidentifyConfig, $inspectConfig, $items);
+     * } finally {
+     *     $dlpServiceClient->close();
+     * }
+     * ```
+     *
+     * @param DeidentifyConfig $deidentifyConfig Configuration for the de-identification of the list of content items.
+     * @param InspectConfig $inspectConfig Configuration for the inspector.
+     * @param ContentItem[] $items The list of items to inspect. Up to 100 are allowed per request.
+     * All items will be treated as text/*.
+     * @param array $optionalArgs {
+     *     Optional.
+     *     @type \Google\GAX\RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\GAX\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\GAX\RetrySettings} for example usage.
+     * }
+     *
+     * @return \Google\Privacy\Dlp\V2beta1\DeidentifyContentResponse
+     *
+     * @throws \Google\GAX\ApiException if the remote call fails
+     * @experimental
+     */
+    public function deidentifyContent($deidentifyConfig, $inspectConfig, $items, $optionalArgs = [])
+    {
+        $request = new DeidentifyContentRequest();
+        $request->setDeidentifyConfig($deidentifyConfig);
+        $request->setInspectConfig($inspectConfig);
+        $request->setItems($items);
+
+        $defaultCallSettings = $this->defaultCallSettings['deidentifyContent'];
+        if (isset($optionalArgs['retrySettings']) && is_array($optionalArgs['retrySettings'])) {
+            $optionalArgs['retrySettings'] = $defaultCallSettings->getRetrySettings()->with(
+                $optionalArgs['retrySettings']
+            );
+        }
+        $mergedSettings = $defaultCallSettings->merge(new CallSettings($optionalArgs));
+        $callable = ApiCallable::createApiCall(
+            $this->dlpServiceStub,
+            'DeidentifyContent',
+            $mergedSettings,
+            $this->descriptors['deidentifyContent']
+        );
+
+        return $callable(
+            $request,
+            [],
+            ['call_credentials_callback' => $this->createCredentialsCallback()]);
+    }
+
+    /**
+     * Schedules a job to compute risk analysis metrics over content in a Google
+     * Cloud Platform repository.
+     *
+     * Sample code:
+     * ```
+     * try {
+     *     $dlpServiceClient = new DlpServiceClient();
+     *     $privacyMetric = new PrivacyMetric();
+     *     $sourceTable = new BigQueryTable();
+     *     $operationResponse = $dlpServiceClient->analyzeDataSourceRisk($privacyMetric, $sourceTable);
+     *     $operationResponse->pollUntilComplete();
+     *     if ($operationResponse->operationSucceeded()) {
+     *       $result = $operationResponse->getResult();
```

Comment:
This is an existing issue, so feel free to address this outside the scope of this PR - but just noticed there are only two spaces for indentation here instead of four.